### PR TITLE
Spring-Core PR

### PR DESCRIPTION
**Pull request by Lasha-Ilashvili**
Added a project and configured service layer

### CODE REVIEW

File: src/test/java/com/internship/gym/data/service/TraineeServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import com.internship.gym.data.model.user.Trainee;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+public class TraineeServiceTest {
```
Comment (vm137) [inline src/test/java/com/internship/gym/data/service/TraineeServiceTest.java:11]: no need for public keyword here and in all other tests, you can remove that

File: src/test/java/com/internship/gym/GymServiceTests.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import org.junit.platform.suite.api.Suite;
+
+@Suite
+@SelectClasses({
+        TraineeServiceTest.class,
+        TrainerServiceTest.class,
+        TrainingServiceTest.class
+})
+public class GymServiceTests {
+}
```
Comment (vm137) [inline src/test/java/com/internship/gym/GymServiceTests.java:16]: in the future, don't check-in empty classes like this

File: src/main/java/com/internship/gym/data/model/training/TrainingType.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package com.internship.gym.data.model.training;
+
+public class TrainingType {
+
+    private String trainingType;
```
Comment (vm137) [inline src/main/java/com/internship/gym/data/model/training/TrainingType.java:5]: It’s confusing to have a class member with the same name, better rename it.

**Review by vm137 — APPROVED** [review]
In general looks good. 
You can fix minor issues I mentioned and merge this PR.

File: src/test/java/com/internship/gym/GymServiceTests.java (hunk 2 above)
Comment (Lasha-Ilashvili) [reply src/test/java/com/internship/gym/GymServiceTests.java:16]: I just used this class in order to run all tests from centralized place (I didn't know how to do it better)

**Pull request merged**